eth/main.cpp

```diff
@@ -773,9 +773,12 @@ int main(int argc, char** argv)
     if (vm.count("help"))
     {
         cout
-                << "Usage eth [OPTIONS]\n"
-                << "Options:\n\n"
-                << "Wallet usage:\n";
+			<< "NAME:\n"
+			<< "   eth - the cpp-ethereum command line interface\n\n"
+			<< "   Copyright 2013-2018 The cpp-ethereum Authors\n\n"
+			<< "USAGE:\n"
+			<< "   eth [options] command [command options] [arguments...]\n\n"
```

Comment:
I don't think this is accurate, we don't really have "commands" as geth does. I'd leave it like `eth [OPTIONS]`